Runtime/Runner/SimvaManager.cs

```diff
@@ -160,7 +160,7 @@ public IAsyncOperation LoginAndSchedule(string token)
                 .Catch(error =>
                 {
                     NotifyLoading(false);
-                    NotifyManagers(error.Message);
+                    NotifyManagers("Failed to Login with this token");
```

Comment:
Add a SimvaPlugin.Instance error log of the error for debugging.

Reply:
fixed